pkg/commands/apply/apply.go

```diff
@@ -148,12 +156,11 @@ func (c *ApplyCommand) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("failed to parse flags: %w", err)
 	}
 
-	parsedArgs := f.Args()
-	if len(parsedArgs) != 1 {
+	if c.flagDir == "" {
```

Comment:
This is usually done in an afterparse function and a better error message is given. See other commands.

Reply:
fixed.